bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -224,12 +224,42 @@ public Collection<WorkItem> prRun(Path scratchPath) {
         // First determine if the current state of the PR has already been checked
         var seedPath = bot.seedStorage().orElse(scratchPath.resolve("seeds"));
         var hostedRepositoryPool = new HostedRepositoryPool(seedPath);
-
-        var census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchPath.resolve("census"), pr,
-                                           bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef()).orElseThrow();
+        CensusInstance census;
         var comments = pr.comments();
         var allReviews = pr.reviews();
         var labels = new HashSet<>(pr.labelNames());
+        try {
+            census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchPath.resolve("census"), pr,
+                    bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef());
+        } catch (MissingJCheckConfException e) {
+            if (bot.confOverrideRepository().isEmpty()) {
+                log.info("No .jcheck/conf found in repo " + bot.repo().name() + ": " + e);
```

Comment:
I think we want SEVERE log messages with the exception included for all the cases here.

Reply:
> I think we want SEVERE log messages with the exception included for all the cases here.

I thought if the jcheck configuration file is not overridden, we just need to tell the user to contact the repo owner and only the repo owner could fix this problem, admin could not. 

For the case that  jcheck configuration file is  overridden, we need to notify the admin because admin could fix this problem.

But it is also reasonable to notify admin in the first case, maybe the admin could contact the repo owner too.

Reply:
Your reasoning is valid, but I think it's better that we get notified as admins too. We need to help repo owners with getting things right, also logging the exception gives us access to the stack trace which can help diagnose further.

Reply:
> Your reasoning is valid, but I think it's better that we get notified as admins too. We need to help repo owners with getting things right, also logging the exception gives us access to the stack trace which can help diagnose further.

Yes, I will change the log!